app/src/main/java/de/xikolo/App.kt

```diff
@@ -102,4 +117,40 @@ class App : Application() {
         CookieManager.getInstance().flush()
     }
 
+
+    @RequiresApi(Build.VERSION_CODES.N_MR1)
+    private fun configureShortcuts() {
+        val recentCoursesObserver = Observer<String> {
+            updateShortcuts()
+        }
+
+        val recentCourses = RecentCoursesStorage()
```

Comment:
This variable is the storage not the courses, so please call it `recentCoursesStorage`.